media-capabilities/decodingInfo.any.js

```diff
@@ -172,6 +186,45 @@ promise_test(t => {
   }));
 }, "Test that decodingInfo rejects if the video configuration contentType has one parameter that isn't codecs");
 
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'file',
+    video: {
+      contentType: 'video/webm',
+      width: 800,
+      height: 600,
+      bitrate: 3000,
+      framerate: 24,
+    },
+  }));
+}, "Test that decodingInfo rejects if the video configuration contentType does not imply a single media codec but has no codecs parameter");
+
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'file',
+    video: {
+      contentType: 'video/webm; codecs="vp09.00.10.08, vp8"',
+      width: 800,
+      height: 600,
+      bitrate: 3000,
+      framerate: '24000/1001',
+    }
+  }));
+}, "Test that decodingInfo() rejects if the video configuration contentType has a codecs parameter that indicates multiple video codecs");
```

Comment:
For consistency with other messages:

```suggestion
}, "Test that decodingInfo rejects if the video configuration contentType has a codecs parameter that indicates multiple video codecs");
```

Reply:
Fixed in https://github.com/web-platform-tests/wpt/pull/49140/commits/56583ab14e348a6daa7614c50ddac447d5ec03e9.